css/mobile.css

```diff
@@ -0,0 +1,386 @@
+  clip-path: polygon(52% 52%, 0% 100%, 100% 100%);
+}
+
+.square-box-4 {
+  width: 79.88px;
+  height: 79.89px;
+}
+
+.semicircle-chat,
+.semicircle-chat-2 {
+  margin-right: 5px;
+  width: 40px;
+  height: 40px;
+  background-color: #e35461;
+  clip-path: polygon(50% 0%, 80% 10%, 100% 35%, 100% 70%, 80% 90%, 50% 100%, 11% 99%, 0% 70%, 0% 35%, 20% 10%);
+}
+
+.semicircle-chat-2 {
+  background-color: #98dede;
+}
+
+.semicircle,
+.semicircle-3 {
+  width: 76px;
+  height: 76px;
+  border-top-right-radius: 50%;
+  border-bottom-right-radius: 50%;
+  align-items: center;
+}
+
+.semicircle-3 {
+  background-color: #e45865;
+  width: 80px;
+  height: 77.94px;
+}
+
+.semicircle-2 {
+  flex-basis: 50%;
+  height: 100%;
+}
+
+.first-semi-circle {
+  border-top-left-radius: 100%;
+}
+
+.second-semi-circle {
+  border-bottom-left-radius: 100%;
+}
+
+.square-box-5 {
+  width: 100%;
+  height: 50%;
+  background: linear-gradient(to top left, #ffff 50%, #98dede 50%);
+}
+
+.triangle-3 {
+  width: 100%;
+  height: 100%;
+  background: #e35461;
+  clip-path: polygon(100% 0, 50% 50%, 100% 100%);
+}
+
+/* Project */
+
+.project-container {
+  margin-top: 50px;
+}
+
+.project {
+  margin-bottom: 32px;
+}
+
+.project-card-title {
+  color: #fff;
+  font-family: "poppins", sans-serif;
+  font-weight: 300;
+}
+
+.project-body {
+  background: #1a2236;
+  padding: 15px;
+}
+
+.skills {
+  margin-top: 16px;
+}
+
+.skills li {
+  background: #98dede;
+  font-size: 10.33px;
+  padding: 6px 10px;
+  font-family: "poppins", sans-serif;
+  text-align: center;
+  list-style: none;
+}
+
+.project-link {
+  border: 1px solid #fff;
+  width: 180px;
+  padding: 10px;
+  background: #1a2236;
+  margin-top: 20px;
+}
+
+.project-link a {
+  color: #fff;
+  text-decoration: none;
+}
+
+.project-svg {
+  vertical-align: middle;
+}
+
+/* about */
+.about-container {
+  margin-top: 56px;
+  padding: 15px;
+}
+
+.about,
+.get-started {
+  font-size: 50px;
+  font-weight: 400;
+  margin-bottom: 20px;
+  font-family: 'Playfair Display', sans-serif;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.about-paragraph {
+  font-size: 14px;
+  margin-right: 16px;
+  color: #1a2236;
+}
+
+.about-link {
+  margin: 30px 0 40px;
+  padding: 10px;
+  text-align: center;
+  background: #1a2236;
+  font-weight: 500;
+}
+
+.about-link a {
+  text-decoration: none;
+  color: #fff;
+  vertical-align: middle;
+}
+
+.about-card {
+  padding: 50px;
+  color: #fff;
+}
+
+.about-cards-1 {
+  background: #1c528a;
+  position: relative;
+}
+
+.about-cards-2 {
+  background: #e35461;
+  position: relative;
+}
+
+.about-cards-3 {
+  position: relative;
+}
+
+.about-card-tech-list {
+  margin-top: 20px;
+}
+
+.about-card-tech-list li {
+  list-style-type: disc;
+  list-style-position: inside;
+}
+
+.hashtag {
+  position: absolute;
+  right: 0;
+  bottom: -7px;
+}
+
+.rectangle-triangle {
+  position: absolute;
+  right: 0;
+  bottom: -7px;
+}
+
+.bulb {
+  position: absolute;
+  right: 0;
+  bottom: -7px;
+}
```

Comment:
These 3 classes has exactly the same CSS properties. It would be better to just make that into one class and re-use it across the elements where needed

Reply:
Ok 
I have grouped the three classes together as a selector and gave it the same CSS properties